src/main/kotlin/dev/ocpd/jsensible/internal/nullability/NullabilityAnnotations.kt

```diff
@@ -10,19 +13,17 @@ import dev.ocpd.jsensible.internal.JavaClassPredicates.anyOf
  */
 internal object NullabilityAnnotations {
 
-    internal fun nullableAnnotations(): DescribedPredicate<JavaAnnotation<*>> = rawType(
-        anyOf(
-            "org.jetbrains.annotations.Nullable",
-            "javax.annotation.Nullable",
-            "org.springframework.lang.Nullable"
-        )
-    ).forSubtype()
-
     internal fun nonNullAnnotations(): DescribedPredicate<JavaAnnotation<*>> = rawType(
         anyOf(
             "org.jetbrains.annotations.NotNull",
             "javax.annotation.Nonnull",
             "org.springframework.lang.NonNull"
         )
     ).forSubtype()
+
+    internal fun nullableAnnotations(): DescribedPredicate<JavaClass> =
+        simpleName("Nullable")
+
+    internal fun springNullableAnnotation(): DescribedPredicate<JavaClass> =
+        assignableTo("org.springframework.lang.Nullable")
```

Comment:
Using `assignableTo` here is quite weird, as I don't think you can extend an annotation class, can you?